models/model.js

```diff
@@ -4,4 +4,22 @@ exports.selectCategories = () => {
     return db.query(`SELECT * FROM categories;`).then(({ rows: topics }) => {
         return topics;
     });
+};
+
+exports.selectReviewById = (id) => {
+    return db
+        .query(`SELECT * FROM reviews WHERE reviews.review_id = $1;`, [id])
+        .then(({ rows }) => {
+            return rows[0];
+        })
+        .then((review) => {
+            if (review === undefined) {
```

Comment:
This is very close, I'd be looking at conditional logic for the row count rather than checking if something is undefined.